core/tasks.py

```diff
@@ -12,40 +13,49 @@
 from chronam.core.models import Batch, OcrDump
 from chronam.core.batch_loader import BatchLoader
 
-logger = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
 
+@task
+def load_page_in_cache(url):
+    try:
+        _logger.debug("Requesting page [%s]" % url)
+        r = requests.get(url)
+        _logger.debug("status %s" % r.status_code)
+        r.raise_for_status() #throw an error if status is not OK
+    except Exception, e:
+        _logger.exception("Unable to request page [%s]: %s" % (url, e))
```

Comment:
In this case, logging makes sense because Celery is running in isolation from however the process was invoked. I would change this to include the status code and make sure that we either don't have `allow_redirects` enabled (I think it's true by default) or that we include both the requested URL and `r.url` so you could see the final URL it was redirected to.